environments/production/inventory.ini

```diff
@@ -331,3 +331,10 @@ hqdjangomanage0.internal-va.commcarehq.org
 
 [django_manage:children]
 hqdjangomanage0
+
+[openvpn]
+10.202.20.194  # ansible_host=18.235.223.2
```

Comment:
Is the comment checked in on purpose?

Reply:
Yeah it is. It's the machine's public IP address, which you occasionally have to use if you get locked out, but is primarily part of the first-time set up of the VPN VM. It's autogenerated by the aws-fill-inventory script. See the comment here for how it's intended to be used in first-time setup: https://github.com/dimagi/commcare-cloud/pull/2222/files#diff-3873a0715ac8c9224b7b1992e21bf79cR107